umh-core/pkg/communicator/api/v2/login.go

```
@@ -0,0 +1,100 @@
+	"github.com/google/uuid"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/communicator/api/v2/http"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/communicator/backend_api_structs"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/communicator/pkg/hash"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/communicator/pkg/tools"
+	"go.uber.org/zap"
+)
+
+// login logs in to the API and returns a JWT token, UUID & the instance name
+func login(token string, insecureTLS bool) (*LoginResponse, error) {
+	//	zap.S().Debugf("Attempting to login with: %s", token)
+	var cookieMap map[string]string
```

golangci-lint (1.64.8)

S1021: should merge variable declaration with assignment on next line

(gosimple)